app/models/category_recommendation.rb

```diff
@@ -0,0 +1,8 @@
+class CategoryRecommendation
+  attr_accessor :category, :media, :id
+  def initialize(params)
```

Comment:
It's optional but if you want to be a little more explicit, you can do this:

```ruby
def initialize(category:, media:, id:)
  @category = category
  @media = media
  @id = id
end
```

Also it looks like `id` is always nil, so maybe you could default it to `category.id`, just for the sake of having an ID? 🤔